packages/esm-patient-common-lib/src/types/index.ts

```diff
@@ -20,10 +20,45 @@ export interface DashboardConfig extends DashboardLinkConfig {
 
 export interface PatientProgram {
   uuid: string;
+  patient?: DisplayMetadata;
+  program: {
+    uuid: string;
+    name: string;
+    allWorkflows: Array<{
+      uuid: string;
+      concept: DisplayMetadata;
+      retired: boolean;
+      states: Array<{}>;
+      links?: Links;
+    }>;
+    concept: {
+      display: string;
+      uuid: string;
+    };
+    links: Links;
+  };
   display: string;
-  patient: OpenmrsResource;
-  program: OpenmrsResource;
   dateEnrolled: string;
-  dateCompleted: string;
-  location: OpenmrsResource;
+  dateCompleted: string | null;
+  location?: {
+    uuid: string;
+    display: string;
+    links: Links;
+  };
+  voided?: boolean;
+  outcome?: null;
+  states?: [];
+  links: Links;
+  resourceVersion?: string;
+}
+
+export type Links = Array<{
+  rel: string;
+  uri: string;
+}>;
```

Comment:
@CynthiaKamau do we have a field `resourceAlias` for Links?

Reply:
Yes, please refer to the rest endpoint response [here](https://rest.openmrs.org/#list-program-enrollment-by-uuid)